src/components/services-view/services-view.controller.ts

```diff
@@ -191,6 +191,10 @@ export class ServicesViewController implements angular.IController {
   private filterChange = (filters: any) => {
     this.filterByCategory(this.ctrl.currentFilter, this.ctrl.currentSubFilter, false);
 
+    // only use filters which have a filter criteria 'value'
+    // prevents applying an empty keyword filter
```

Comment:
Can we link to an upstream issue or PR in Patternfly with a TODO to remove this later? Let me know if you'd like me to open an issue.